File: app/src/test/java/com/felipecsl/elifut/TestElifutApplication.java
package com.felipecsl.elifut;

public class TestElifutApplication extends ElifutApplication {

  @Override public void onCreate() {
    // Never call super() so we don't initialize Stetho
    component = TestElifutComponent.Initializer.init(this);
  }

  public TestElifutComponent testComponent() {
    return (TestElifutComponent) component;
  }
}

File: app/src/test/java/com/felipecsl/elifut/TestElifutApplication.kt
package com.felipecsl.elifut

class TestElifutApplication : ElifutApplication() {

  override fun onCreate() {
    // Never call super() so we don't initialize Stetho
    component = TestElifutComponent.Initializer.init(this)
  }

  fun testComponent(): TestElifutComponent {
    return component as TestElifutComponent
  }
}
